### PR TITLE
Width and height swapped when creating test image in data/synthetic/create_test_image_2d

### PR DESCRIPTION
**Pull request by OeslleLucena**
Signed-off-by: OeslleLucena <oeslle.lucena@gmail.com>

Fixes https://github.com/Project-MONAI/MONAI/issues/5373.

### Description
Change the `synthetic.py` script to have the (height, width, depth) format.
Edit all the tests that uses `create_test_image_3d` to match the new format.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [x] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Review by wyli — DISMISSED** [review]
Thanks, it looks good to me.

File: monai/data/synthetic.py [hunk 1]
```diff
@@ -134,19 +134,19 @@ def create_test_image_3d(
...
+    min_size = min(height, width, depth)
     if min_size <= 2 * rad_max:
         raise ValueError("the minimal size of the image should be larger than `2 * rad_max`.")
 
-    image = np.zeros((width, height, depth))
+    image = np.zeros((height, width, depth))
     rs: np.random.RandomState = np.random.random.__self__ if random_state is None else random_state  # type: ignore
 
     for _ in range(num_objs):
-        x = rs.randint(rad_max, width - rad_max)
```
Comment (wyli) [inline monai/data/synthetic.py:145]: this changes of randomisations break some integration tests...

```
[2022-12-14T13:55:10.871Z] ======================================================================
[2022-12-14T13:55:10.871Z] ERROR: test_training (__main__.TestIntegrationSlidingWindow)
[2022-12-14T13:55:10.871Z] ----------------------------------------------------------------------
[2022-12-14T13:55:10.871Z] AssertionError: 
[2022-12-14T13:55:10.871Z] Not equal to tolerance rtol=1e-07, atol=0
[2022-12-14T13:55:10.871Z] 
[2022-12-14T13:55:10.871Z] Mismatched elements: 1 / 1 (100%)
[2022-12-14T13:55:10.871Z] Max absolute difference: 992.
[2022-12-14T13:55:10.871Z] Max relative difference: 0.02950537
[2022-12-14T13:55:10.871Z]  x: array(34613.)
[2022-12-14T13:55:10.871Z]  y: array(33621)
[2022-12-14T13:55:10.871Z] 
[2022-12-14T13:55:10.871Z] The above exception was the direct cause of the following exception:
[2022-12-14T13:55:10.871Z] 
[2022-12-14T13:55:10.871Z] Traceback (most recent call last):
[2022-12-14T13:55:10.871Z]   File "/home/jenkins/agent/workspace/Monai-pytorch-versions/tests/utils.py", line 612, in _wrapper
[2022-12-14T13:55:10.871Z]     raise RuntimeError(res.traceback) from res
[2022-12-14T13:55:10.871Z] RuntimeError: Traceback (most recent call last):
[2022-12-14T13:55:10.871Z]   File "/home/jenkins/agent/workspace/Monai-pytorch-versions/tests/utils.py", line 563, in run_process
[2022-12-14T13:55:10.871Z]     output = func(*args, **kwargs)
[2022-12-14T13:55:10.871Z]   File "/home/jenkins/agent/workspace/Monai-pytorch-versions/tests/utils.py", line 644, in _call_original_func
[2022-12-14T13:55:10.871Z]     return f(*args, **kwargs)
[2022-12-14T13:55:10.871Z]   File "/home/jenkins/agent/workspace/Monai-pytorch-versions/tests/test_integration_sliding_window.py", line 90, in test_training
[2022-12-14T13:55:10.871Z]     np.testing.assert_allclose(np.sum(output_image), 33621)
[2022-12-14T13:55:10.871Z]   File "/usr/local/lib/python3.8/dist-packages/numpy/testing/_private/utils.py", line 1527, in assert_allclose
[2022-12-14T13:55:10.871Z]     assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
[2022-12-14T13:55:10.871Z]   File "/usr/local/lib/python3.8/dist-packages/numpy/testing/_private/utils.py", line 844, in assert_array_compare
[2022-12-14T13:55:10.871Z]     raise AssertionError(msg)
[2022-12-14T13:55:10.871Z] AssertionError: 
[2022-12-14T13:55:10.871Z] Not equal to tolerance rtol=1e-07, atol=0
[2022-12-14T13:55:10.871Z] 
[2022-12-14T13:55:10.871Z] Mismatched elements: 1 / 1 (100%)
[2022-12-14T13:55:10.871Z] Max absolute difference: 992.
[2022-12-14T13:55:10.871Z] Max relative difference: 0.02950537
[2022-12-14T13:55:10.871Z]  x: array(34613.)
[2022-12-14T13:55:10.871Z]  y: array(33621)
[2022-12-14T13:55:10.871Z] 
```

Comment (OeslleLucena) [reply monai/data/synthetic.py:145]: I edited this line to `x = rs.randint(rad_max, height - rad_max)`, why it shows `width`?

Comment (wyli) [reply monai/data/synthetic.py:145]: this is to check that the workflow runs deterministically, repeating the tests always generates the same number...

Comment (wyli) [reply monai/data/synthetic.py:145]: I see, thanks! I'll trigger the tests and merge this if all works fine

**Review by wyli — APPROVED** [review]

**Pull request merged**